x-pack/test/plugin_api_perf/config.js

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Could use a README.md in here to describe how to use this; I assume since the top-level describe is a "skip", to run this you would manually change that to just a "describe", and run FTS/FTR with this config?  Which is the "manual" part of the test.

Reply:
done :)